src/WebJobs.Script/Config/ScriptHostOptionsSetup.cs

```diff
@@ -135,129 +142,5 @@ private void ConfigureLanguageWorkers(IConfigurationSection rootConfig, ScriptJo
-        // Also noticed a lot of discrepancy between the original logic and the documentation and schema. We need to address
-        // that. The current implemenation will match the original, which means that schema and docs will need to be udpated.
-        //internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration scriptConfig, ILogger logger = null)
-        //{
-        //    var hostConfig = scriptConfig.HostOptions;
-
-        //    hostConfig.HostConfigMetadata = config;
-
-        //    JArray functions = (JArray)config["functions"];
-        //    if (functions != null && functions.Count > 0)
-        //    {
-        //        scriptConfig.Functions = new Collection<string>();
-        //        foreach (var function in functions)
-        //        {
-        //            scriptConfig.Functions.Add((string)function);
-        //        }
-        //    }
-        //    else
-        //    {
-        //        scriptConfig.Functions = null;
-        //    }
-
-        //    // We may already have a host id, but the one from the JSON takes precedence
-        //    JToken hostId = (JToken)config["id"];
-        //    if (hostId != null)
-        //    {
-        //        hostConfig.HostId = (string)hostId;
-        //    }
-
-        //    // Default AllowHostPartialStartup to true, but allow it
-        //    // to be overridden by config
-        //    hostConfig.AllowPartialHostStartup = true;
-        //    JToken allowPartialHostStartup = (JToken)config["allowPartialHostStartup"];
-        //    if (allowPartialHostStartup != null && allowPartialHostStartup.Type == JTokenType.Boolean)
-        //    {
-        //        hostConfig.AllowPartialHostStartup = (bool)allowPartialHostStartup;
-        //    }
-
-        //    JToken fileWatchingEnabled = (JToken)config["fileWatchingEnabled"];
-        //    if (fileWatchingEnabled != null && fileWatchingEnabled.Type == JTokenType.Boolean)
-        //    {
-        //        scriptConfig.FileWatchingEnabled = (bool)fileWatchingEnabled;
-        //    }
-
-        //    // Configure the set of watched directories, adding the standard built in
-        //    // set to any the user may have specified
-        //    if (scriptConfig.WatchDirectories == null)
-        //    {
-        //        scriptConfig.WatchDirectories = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
-        //    }
-        //    scriptConfig.WatchDirectories.Add("node_modules");
-        //    JToken watchDirectories = config["watchDirectories"];
-        //    if (watchDirectories != null && watchDirectories.Type == JTokenType.Array)
-        //    {
-        //        foreach (JToken directory in watchDirectories.Where(p => p.Type == JTokenType.String))
-        //        {
-        //            scriptConfig.WatchDirectories.Add((string)directory);
-        //        }
-        //    }
-
-        //    JToken nugetFallbackFolder = config["nugetFallbackFolder"];
```

Comment:
Automatically handled by binding to configuration.

Reply:
It still doesn't look like the fallback folder is supported, right?

Reply:
It is. This will be set as part of the bind call.